src/mount.rs

```diff
@@ -0,0 +1,655 @@
+use lru::LruCache;
```

Comment:
Can we call this `mount/projfs.rs` or something? We can have a separate FUSE mount on Linux.